src/predict_request_validation_utils.cpp

```diff
@@ -580,6 +726,19 @@ Status RequestValidator<KFSRequest, KFSTensorInputProto, KFSInputTensorIteratorT
     }
     return StatusCode::OK;
 }
+template <>
```

Comment:
C-API does not use protobuf so it would be good to not use proto as a parameter name to not confuse